src/data/finset/prod.lean

```diff
@@ -71,6 +71,10 @@ product_subset_product hs (subset.refl _)
 lemma product_subset_product_right (ht : t ⊆ t') : s ×ˢ t ⊆ s ×ˢ t' :=
 product_subset_product (subset.refl _) ht
 
+lemma map_swap_product (s : finset α) (t : finset β) :
```

Comment:
Why not `simp`?

Reply:
because I'm not sure of the shape of the LHS

Reply:
I don't follow; why would that matter? If the image lemma is simp, the map one should be too

Reply:
I mean that I don't think it will ever fire.